src/utils.ts

```diff
@@ -234,6 +245,19 @@ export namespace Utils {
           "The transaction has operations that are unrecognized",
         );
       }
+      if (op.value === undefined) {
```

Comment:
Hi @leighmcculloch @JakeUrban, I have some confusion about this, does the SEP-10 protocol specify that the values in other ManageData Operation cannot be null?

Reply:
That's a good point, it does not specify this and shouldn't reject a challenge transaction because of this.

Reply:
Nice catch.I'll fix it today, before release.